### DISCUSSION TITLE
Choose color (+symbol) for all waypoints in a set

### DISCUSSION THREAD

**Discussion by Shohreh** [organicmaps/organicmaps#1170] [Ideas]
Hello,

Although I suggested this features years ago to the Maps.me team, it was never implemented.

It'd be useful to be able to set the color of a group of bookmarks in one go. It'd be even better if users could also choose a symbol instead of just a color.

When I travel, I create different sets (housing, sights, restaurants, etc.), each in a different color so as to be able to show/hide each one independently.

Since Maps.me doesn't support setting the color once at the top level, I had to write a script to add an element to each waypoint, eg. &lt;styleUrl&gt;#placemark-yellow&lt;/styleUrl&gt;.

Cheers,

**Comment by biodranik** [comment]
That is a good idea. Implementing it is only the question of time and our limited resources. With the help of the community it may be faster.

**Comment by Shohreh** [comment]
Incidently, through styling, is it possible to set the color for all bookmarks in a given KML file, instead of adding it to each and every waypoint?

&lt;Placemark&gt;&lt;styleUrl&gt;#placemark-yellow&lt;/styleUrl&gt;

That would be simpler.

--
Edit: OM/Maps.me doesn't seem to support it: The waypoint is still displayed in the default red, although I chose yellow (ff00ffff):

```
<?xml version="1.0" encoding="UTF-8"?>
<kml xmlns="http://www.opengis.net/kml/2.2" xmlns:gx="http://www.google.com/kml/ext/2.2">
  <Document>
    <name>Test waypoints color</name>
		<Style id="waypoint">
				<IconStyle>
					 <color>ff00ffff</color>
				</IconStyle>
		</Style>    
    <Folder>
      <name>Waypoints</name>
      <Placemark>
        <styleUrl>#waypoint</styleUrl>
        <Point>
          <coordinates>0.170900,45.271800</coordinates>
        </Point>
      </Placemark>
    </Folder>
  </Document>
</kml>
```
